### ISSUE TITLE
Multiple tds per tr

### ISSUE THREAD

**Issue by arasmussen** [chromakode/react-html-email#11]
Give your API components, I don't see a way to create multiple items in a single row. What if I'd like to have 2 boxes of equal size, side by side. Should I fall back to divs or just render raw tds with width="50%"?

**Comment by Roilan** [comment]
Hey @arasmussen,

I've been using raw tds since I've ran into the same problem. I agree there should be an `ItemContainer` component that allows you to render `tds` inside.

**Comment by chromakode** [comment]
Good point! Speaking in general, the components provided by this library are intended as a jumping off point -- if you need to get fancy, feel free to use them as an example and create your own `<td>` / `<tr>` nodes. That being said, I think moving to an `<ItemContainer>`/`<Item>` combo sounds good; we could bump to v2 for that change.

**Comment by pwmckenna** [comment]
Didn't see this until after submitting a pr, so I don't know if it aligns with what @arasmussen wanted. https://github.com/chromakode/react-html-email/pull/21

**Comment by acomito** [comment]
Yes! I have a situation where I need 2 columns as well.
